### PR TITLE
Downloads: Style: monitor demo as a subsection of Monitor

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 09.Downloads/docs.md [hunk 1]
```diff
@@ -408,7 +408,7 @@ dpkg -i mender-monitor_master-1+debian+buster_all.deb
 apt --fix-broken -y install
 ```
 
-# Mender monitor demo
+### Mender monitor demo
```
Comment (merlin-northern) [inline 09.Downloads/docs.md:411]: suggestion: the main section reads: `## Monitor` so maybe something like `### Demo` or `### Monitor Demo`

**Review by merlin-northern — CHANGES_REQUESTED** [review]

**Review by merlin-northern — APPROVED** [review]
pre-approving as the comment is a suggestion.

**Review by oleorhagen — APPROVED** [review]
Approving, but agreeing with Piotr's comment

File: 09.Downloads/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:411]: What about `### Demo monitors` ?

Comment (merlin-northern) [reply 09.Downloads/docs.md:411]: makes sense to me as you already call them that below.

**Review by merlin-northern — APPROVED** [review]

**Comment by lluiscampos** [comment]
@mender-test-bot cherry-pick to:
* 3.2.x
* hosted

**Pull request merged**